app/views/layouts/_footer_navbar.html.erb

```diff
@@ -9,7 +9,7 @@
     <li><%= link_to 'News', news_messages_path %></li>
     <li><%= link_to 'Administration', admin_control_path %></li>
     <% if current_user && current_user.super_admin? %>
-    <li><%= link_to 'Super Administration', users_path %></li>
+    <li><%= link_to 'User Administration', users_path %></li>
```

Comment:
Yeah, the permission is solid, I was only renaming the link for clarity. I didn't want one of the rare Super Admins to not know what was behind Door #2. It may make sense to change back at a future time when Super Admins can do more than just change Users.